docs/regmap/adi_regmap_spi_engine.txt

```diff
@@ -55,7 +55,7 @@ SCRATCH
 ENDREG
 
 FIELD
-[31:0] 0x00000000
+[31:0] 0x0
 SCRATCH
 RW
 Scratch register useful for debug.
```

Comment:
DATA_WIDTH register description is not correct and not complete. Value should not be 0x8, register is dependent on params

https://github.com/analogdevicesinc/hdl/blob/9ba4c66c63792523bd6ab3b7b475a540aebab122/library/spi_engine/axi_spi_engine/axi_spi_engine.v#L348

Reply:
Done here: https://github.com/analogdevicesinc/hdl/pull/1294/commits/9c5e3b37e5973b952a7d83267d5709effa4d17b8